@emersonknapp It seems need to add `action_msgs` in package.xml as dependencies.
```
13:40:50 --- stderr: rosbag2_storage_mcap_testdata
13:40:50 CMake Error at /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:147 (message):
13:40:50   Unable to generate action interface for 'action/BasicAction.action'.  In
13:40:50   order to generate action interfaces you must add a depend tag for
13:40:50   'action_msgs' in your package.xml.
13:40:50 Call Stack (most recent call first):
13:40:50   CMakeLists.txt:11 (rosidl_generate_interfaces)
```

--- a/rosbag2_storage_mcap/src/message_definition_cache.cpp
+++ b/rosbag2_storage_mcap/src/message_definition_cache.cpp
@@ -32,7 +32,6 @@
 
 namespace rosbag2_storage_mcap::internal
 {
-
 /// A type name did not match expectations, so a definition could not be looked for.
 class TypenameNotUnderstoodError : public std::exception
 {

--- a/rosbag2_storage_mcap_testdata/package.xml
+++ b/rosbag2_storage_mcap_testdata/package.xml
@@ -12,6 +12,8 @@
   <buildtool_depend>ament_cmake</buildtool_depend>
   <buildtool_depend>rosidl_default_generators</buildtool_depend>
 
+  <depend>action_msgs</depend>
+
   <test_depend>ament_lint_auto</test_depend>
   <test_depend>ament_lint_common</test_depend>
 

